packages/backend/src/clients/Google/GoogleDriveClient.ts

```diff
@@ -50,6 +50,34 @@ export class GoogleDriveClient {
         });
     }
 
+    private static async changeTabTitle(
+        sheets: any,
```

Comment:
Maybe this was answered in previous PRs, but was wondering why we're not referencing Google's types from `googleapis` - so this one would be:
```ts
sheets: sheets_v4.Sheets
```

I had to update the import to `import { google, sheets_v4 } from 'googleapis';` so I could get typescript to show me the errors btw 👍

Reply:
Yes of course, great suggestion, I'll fix the types , but perhaps not in this PR but on the main PR or maybe another PR.

Reply:
sounds good! thanks 👍